docs/management/admin/response-actions.asciidoc

```diff
@@ -111,6 +111,29 @@ TIP: You can use the <<use-osquery,Osquery manager integration>> to query a host
 When {elastic-defend} prevents file activity due to <<malware-protection,malware prevention>>, the file is quarantined on the host and a malware prevention alert is created. To retrieve this file with `get-file`, copy the path from the alert's *Quarantined file path* field (`file.Ext.quarantine_path`), which appears under *Highlighted fields* in the alert details flyout. Then paste the value into the `--path` parameter. 
 ====
 
+=== `execute`
+
+Run a shell command on the host. The command's output and any errors are returned in the response console, up to 2000 characters. The complete output (stdout and stderr) are also saved to a downloadable .zip archive (password: `elastic`). Use these parameters:
+
+* `--command` : (Required) A shell command to run on the host. The command must be supported by `bash` for Linux and macOS hosts, and `cmd.exe` for Windows.
++
+[NOTE]
+====
+* Multiple consecutive dashes in the value must be escaped. For example, to represent a directory named `/opt/directory--name`, use the following: `/opt/directory\-\-name`.
+
+* You can use quotation marks without escaping. For example: +
+`execute --command "cd "C:\Program Files\directory""`
+
+====
+
+* `--timeout` : (Optional) How long the host should wait for the command to complete. Use `h` for hours, `m` for minutes, `s` for seconds (for example, `2s` is two seconds). If no timeout is given, it defaults to four hours.
+
+Required privilege: *Execute Operations*
+
+Example: `execute --command "ls -al" --timeout 2s --comment "Get list of all files"`
+
+WARNING: This response action runs commands on the host using the same user account running the {elastic-defend} integration, which normally has full control over the system. Be careful with any commands that could cause irrevocable changes.
```

Comment:
I think this changes the meaning somewhat -- this edit says that the host is using the account, but the host doesn't really _do_ anything, it's just the place/resource where things happen. We might just be able to omit "on the host" to make it clearer that the response action is using the account, not the host.

> Also, do we need to specify the integration here -- or just Elastic Defend as a whole?

Not sure what you mean; `Elastic Defend` _**is**_ the integration. Did you mean `Elastic Endpoint`? Because it might be more accurate to say that the user account is running Endpoint. @caitlinbetz @kevinlog Is that correct?

```suggestion
WARNING: This response action runs commands using the same user account that is running {elastic-endpoint}, which normally has full control over the system. Be careful with any commands that could cause irrevocable changes.
```